Solutions/Marain.ContentManagement.Tenancy.Abstractions/Marain/Cms/ITenantedContentStoreFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="ITenantedContentStoreFactory.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.Cms
+{
+    using System.Threading.Tasks;
+    using Corvus.Tenancy;
+
+    /// <summary>
+    /// Interface for a factory that knows how to construct a content factory for
```

Comment:
Obtain, rather than construct? (It might construct it, but equally, it might just return one it knows already exists.)

Reply:
Truefax.